intel-sgx/fortanix-sgx-tools/src/bin/ftxsgx-runner.rs

```diff
@@ -47,6 +48,30 @@ fn catch_sigbus() {
     }
 }
 
+#[cfg(unix)]
+fn catch_sigusr1() {
```

Comment:
This shouldn't be in ftxsgx-runner. Also, I think this flow should only be used when using the VDSO. This function should also check if it's in the VDSO or somewhere else.

Reply:
moved to enclave_runner/mod.rs, conditionally register handler for vdso case only